safe/note.go

```diff
@@ -25,14 +25,14 @@ func (n *notesManager) Edit(name string) error {
 	if len(name) == 0 {
 		return errors.New("Empty note name specified")
 	}
-	note, exists := n.Notes[name]
+	nt, exists := n.Notes[name]
```

Comment:
Why rename `note` to `nt`?

Reply:
Because `note` is already in use: https://github.com/leonklingele/pick/blob/8a5565ffeb4f1ebfc685c0160695dc7cdf897bac/safe/note.go#L62